### DISCUSSION TITLE
Emacs – supported out of the box (via Diary Mode)

### DISCUSSION THREAD

**Discussion by avillafiorita** [jotaen/klog#309] [Show and tell]
Hi, 
first of all, thanks, Klog's really nice.

Emacs supports Klog out of the box, through Diary Mode. The Klog format is, in fact, compatible with that of the "fancy" diary and the diary commands can be used to operate on Klog files (show day, add entry, export to iCalendar, ...).

More information here: https://ict4g.net/adolfo/notes/emacs/klog-and-emacs-diary.html (feel free to copy/reuse/adapt as you see fit).

**Comment by jotaen** [comment]
Thank you for your kind words, glad that you like klog!

What a cool coincidence that klog is compatible with Emacs diary mode! Thanks for the detailed write-up, that will surely be helpful for other Emacs users. I’ve added “Diary Mode” to the discussion title, to make this discussion thread easier to discover / search for.

**Reply by avillafiorita** [reply]
Excellent, thanks again!
